src/Microsoft.Tye.Core/DockerfileGenerator.cs

```diff
@@ -138,6 +141,9 @@ public static void ApplyContainerDefaults(ApplicationBuilder application, Docker
             }
 
             container.ImageTag ??= "latest";
+
+            // Disable color in the logs
+            project.EnvironmentVariables.Add(new EnvironmentVariableBuilder("DOTNET_LOGGING__CONSOLE__DISABLECOLORS") { Value = "true" });
```

Comment:
This shouldn't be added for all docker files, just .NET projects.

Reply:
Ah yeah that's pretty obvious now, will fix